plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/FastXSLTMediatorPropertiesEditionPartForm.java

```diff
@@ -812,6 +810,19 @@ public void focusGained(FocusEvent e) {
         dynamicSchemaKeyElements = new Control[] { dynamicSchemaKeyLabel, dynamicSchemaKeyText, dynamicSchemaKeyHelp };
         return parent;
     }
+    
+    private void openNamespacedPropertyEditor(final Composite parent) {
+        EEFNameSpacedPropertyEditorDialog nspd = new EEFNameSpacedPropertyEditorDialog(parent.getShell(),
+                SWT.NULL, fastXsltDynamicSchemaKey);
+        // valueExpression.setPropertyValue(valueExpressionText.getText());
```

Comment:
Remove the commented line.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/commit/8bf90d13170529d3776b1699ed2b01a4b8b110d9